Source/relay/TourGuide/Quests/Level 12.ash

```diff
@@ -187,9 +187,19 @@ void QLevel12GenerateTasksSidequests(ChecklistEntry [int] task_entries, Checklis
         modifiers.listAppend("+meat");
         
         string [int] tasks;
-        int ncs_seen = $location[McMillicancuddy's Barn].noncombatTurnsAttemptedInLocation();
+
```

Comment:
huh what caused you to find this one -- you're right, this is a good switch now that there's so many ways to accidentally get NCs into the location (like cleaver, or turtles, etc) but this is wild. were you doing ducks???

Reply:
(to be clear, again, good change! just curious how you even noticed it lmao)

Reply:
I was doing ducks in 2CRS, I think (it blocks nuns), and my parka landed one too many NCs into the barn.